plugins/modules/instance.py

```diff
@@ -74,9 +74,6 @@
       - If machine is already in deployed state, I(deploy_params) will be ignored. Machine needs to be released first for I(deploy_params) to apply
     type: dict
     suboptions:
-      osystem:
-        description: The OS the machine will use.
-        type: str
       distro_series:
         description: The OS release the machine will use.
```

Comment:
`osystem` was removed not to default to Ubuntu but since this parameter was falsely used for a long time. It has no effect as per: https://bugs.launchpad.net/maas/+bug/1923315.

`distro_series` is not defaulting to anything inside ansible. Please take a look [here](https://github.com/maas/ansible-collection/pull/32/files#diff-c3eabaed303c30ec362f31e46892241db96597291675ec94c9c19ea9cf1df1caR373) to confirm this. Instead, it is defaulting to MAAS default, found in http://_ip_:5240/MAAS/r/settings/configuration/deploy. Finally, if you need e.g. centos you can use distro_series `centos/6`

Reply:
ah, perfect